src/test/groovy/org/gradle/android/TestVersions.groovy

```diff
@@ -41,4 +41,21 @@ class TestVersions {
         def minorVersions = allCandidateTestVersions.keySet().collect { "${it.major}.${it.minor}" }
         return minorVersions.collect { getLatestVersionForAndroid(it) }
     }
+
+    static List<String> supportedKotlinVersions = ["1.3.72", "1.5.0"]
+
+    static VersionNumber oldestSupportedKotlinVersion() {
+        return VersionNumber.parse(supportedKotlinVersions.first())
+    }
+
+    static VersionNumber latestSupportedKotlinVersion() {
+        return VersionNumber.parse(supportedKotlinVersions.last())
+    }
+
+    static VersionNumber latestKotlinVersionForGradleVersion(GradleVersion gradleVersion) {
```

Comment:
@ghale I added this since KGP 1.5 doesn't support Gradle versions lower than 6.1